HIP/hip-412.md

```diff
@@ -213,7 +233,152 @@ Note that mime types for directories are not uniformly defined. Some IPFS CIDs p
 
```

Comment:
Update URI Formatting to disallow 'http' protocols as acceptable

`For resources that are on the world wide web, the standard https protocols are acceptable. Ie. https://www.example.org/image/file.jpg`